src/org/zaproxy/zap/extension/ascanrulesBeta/InsecureHTTPMethod.java

```diff
@@ -62,7 +62,10 @@
 			{
 			"TRACE",
 			"TRACK",
-			"CONNECT"
+			"CONNECT",
+			"PUT",
+			"DELETE",
```

Comment:
There should probably also be additional checks around PUT and DELETE. For example: If DELETE is possible put results in a 400 or 500 condition then is it actually a vulnerability? Or if it's available but only when authenticated?